storage/src/main/java/tech/pegasys/artemis/storage/ChainStorageServer.java

```diff
@@ -36,8 +36,9 @@
   private final EventBus eventBus;
 
   public ChainStorageServer(EventBus eventBus, ArtemisConfiguration config) {
+    File databaseStoragePath = new File(config.getDataPath() + "/db");
```

Comment:
I was expecting that the version number would be in dataPath and then MapDB's stuff would be under `/db`. I guess that means `ChainStorageServer` would be responsible for checking the version rather than `MapDbDatabase`.  That would also fit with the idea that we might migrate from MapDb to some other storage system in the future.

Reply:
will be committing this shortly, after our discussion i think this bit is resolved.